documentation/meeting-logs/2023-10-23.md

```diff
@@ -0,0 +1,111 @@
+# Standup Meeting 5
+
+2023-10-23
+Sprint 2
+
+Meeting duration: 2 hour
+
+## Attendees
+
+All group members
+
+## Notes
+
+Updates:
+
+- Dan
+  - Made two new endpoints for accessing it separately
+    - (<https://github.com/BarbzCodez/Spendr/issues/52>)
+  - presentation for friday
+- Victoria
+  - cleaned up front end
+  - completed the following, which was discussed in the past meeting
+    - Login: (<https://github.com/BarbzCodez/Spendr/issues/21>)
+    - Expense History List: (<https://github.com/BarbzCodez/Spendr/issues/27>)
+    - Expense History: (<https://github.com/BarbzCodez/Spendr/issues/28>)
+    - Add Expense: (<https://github.com/BarbzCodez/Spendr/issues/26>)
+    - Edit Expense: (<https://github.com/BarbzCodez/Spendr/issues/60>)
+    - Delete Expense: (<https://github.com/BarbzCodez/Spendr/issues/61>)
+- Ethan
+  - Finished up the prisma
+    - (<https://github.com/BarbzCodez/Spendr/issues/29>)
+  - fixed the middle ware logging bug pop up
+  - May have to change the testing because it cant go through the pipeline
+  - Changing to Morgan, dont have to test it because it is considered middle ware
+- Mark
+  - CD pipeline is completed, PR is up
+    - (<https://github.com/BarbzCodez/Spendr/issues/100>)
+  - Starting working on the budget api
+- Barbra
+  - Finished user settings
+    - (<https://github.com/BarbzCodez/Spendr/issues/23>)
+  - integrated front work with the api for user settings
```

Comment:
```suggestion
- Barbara
  - Finished Budgets UI and middleware with the api, waiting for Delete and Edit api to be done
  - Bug fix: Hide the expense table
  - Cleaned off Documentation
```

Reply:
I left in the link because we need to have them linked. Even if it looks ugly.